napalm/eos.py

```diff
@@ -484,3 +484,18 @@ def get_lldp_neighbors_detail(self, interface = ''):
                 )
 
         return lldp_neighbors_out
+
+    def cli(self, command = None):
+
+        if type(command) is not str:
```

Comment:
Why a string? Wouldn't be better to allow a list of commands instead of just a single command?